MANIFEST.in

```diff
@@ -5,7 +5,7 @@ include README.md
 
 include .bumpversion.cfg
 include .codespellrc
-include .flake8
+include pyproject.toml
```

Comment:
I would make it a ruff.toml unless we also replace setup.py and use pyproject.toml for building as well.

Reply:
Running `setup.py` is now deprecated as of the current version of pip so will run `setuptools-py2cfg` and then `ini2toml` to migrate everything to `pyproject.toml` but let's do that also in a separate PR.